openmetadata-ui/src/main/resources/ui/src/pages/RolesPage/AddRuleModal.tsx

```diff
@@ -0,0 +1,175 @@
+              className="tw-modal-title tw-text-grey-body"
+              data-testid="header">
+              {header}
+            </p>
+          </div>
+          <div className="tw-modal-body">
+            <div className="tw-mb-4">
+              <label className="tw-form-label required-field">Name</label>
+              <input
+                autoComplete="off"
+                className="tw-text-sm tw-appearance-none tw-border tw-border-main
+                tw-rounded tw-w-full tw-py-2 tw-px-3 tw-text-grey-body  tw-leading-tight
+                focus:tw-outline-none focus:tw-border-focus hover:tw-border-hover tw-h-10"
+                name="name"
+                placeholder="Name"
+                type="text"
+                value={data.name}
+                onChange={onChangeHadler}
+              />
+              {errorData?.name && errorMsg(errorData.name)}
+            </div>
+            <div className="tw-grid tw-grid-cols-2 tw-gap-2">
+              {!isUndefined(initialData.operation) && (
+                <div className="tw-mb-4">
+                  <label className="tw-form-label required-field">
+                    Select Operation
+                  </label>
+                  <select
+                    required
+                    className="tw-text-sm tw-appearance-none tw-border tw-border-main
+                tw-rounded tw-w-full tw-py-2 tw-px-3 tw-text-grey-body  tw-leading-tight
+                focus:tw-outline-none focus:tw-border-focus hover:tw-border-hover tw-h-10 tw-bg-white"
+                    name="operation"
+                    value={data.operation}
+                    onChange={onChangeHadler}>
+                    <option value={Operation.UpdateDescription}>
+                      Update Description
+                    </option>
+                    <option value={Operation.UpdateOwner}>Update Owner</option>
+                    <option value={Operation.UpdateTags}>Update Tags</option>
+                    <option value={Operation.SuggestDescription}>
+                      Suggest Description
+                    </option>
+                    <option value={Operation.SuggestTags}>Suggest Tags</option>
+                  </select>
+                  {errorData?.operation && errorMsg(errorData.operation)}
+                </div>
+              )}
+              <div className="tw-mb-4">
+                <label className="tw-form-label">Access</label>
+                <select
+                  required
+                  className="tw-text-sm tw-appearance-none tw-border tw-border-main
+                tw-rounded tw-w-full tw-py-2 tw-px-3 tw-text-grey-body  tw-leading-tight
+                focus:tw-outline-none focus:tw-border-focus hover:tw-border-hover tw-h-10 tw-bg-white"
+                  name="access"
+                  value={access}
+                  onChange={(e) => setAccess(e.target.value as RuleAccess)}>
+                  <option value={RuleAccess.ALLOW}>Allow</option>
```

Comment:
I'm wondering if we should display **ALLOW** or **DENY** in upper case with bold